src/main/resources/view/HelpWindow.fxml

```diff
@@ -19,26 +18,160 @@
-        </opaqueInsets>
-        <padding>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </padding>
-      </HBox>
+      <HBox fx:id="helpMessageContainer" alignment="CENTER" prefHeight="600.0" prefWidth="1000.0" spacing="10.0">
+            <children>
+               <VBox prefHeight="600.0" prefWidth="1000.0">
+                  <children>
+                     <HBox prefHeight="600.0" prefWidth="1000.0">
+                        <children>
+                           <VBox fx:id="firstHalfCommands" prefHeight="299.0" prefWidth="500.0" style="-fx-border-color: white;">
+                              <children>
+                                  <HBox prefHeight="48.0" prefWidth="100.0" spacing="170.0">
+                                      <Label style="-fx-padding: 15 2 5 50;" text="Command Format" textFill="WHITE">
+                                         <font>
+                                             <Font size="15.0" />
+                                         </font>
+                                      </Label>
+                                      <Label style="-fx-padding: 15 0 5 0;" text="Description" textFill="WHITE">
+                                         <font>
+                                             <Font size="15.0" />
+                                         </font>
+                                      </Label>
+                                  </HBox>
+                                  <HBox prefHeight="37.0" prefWidth="495.0">
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T add m/MODULE d/DESCRIPTION" textFill="WHITE" />
+                                      <Label style="-fx-padding: 0 0 0 100;" text="Adds the task to the task list" textFill="WHITE" />
+                                  </HBox>
+                                  <HBox prefHeight="35.0" prefWidth="495.0">
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T del INDEX" textFill="WHITE" />
+                                      <Label prefHeight="17.0" prefWidth="411.0" style="-fx-padding: 0 0 0 222;" text="Deletes the task in the task list" textFill="WHITE" />
+                                  </HBox>
+                                  <HBox prefHeight="38.0" prefWidth="495.0">
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T edit INDEX [m/MODULE] [d/DESCRIPTION]" textFill="WHITE" />
+                                      <Label style="-fx-padding: 0 0 0 43;" text="Edits the task in the task list" textFill="WHITE" />
+                                  </HBox>
+                                  <HBox prefHeight="36.0" prefWidth="495.0">
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T list" textFill="WHITE" />
+                                      <Label style="-fx-padding: 0 0 0 260;" text="Lists the task in the task list" textFill="WHITE" />
```

Comment:
Should this be 'tasks'. Same for sorting below.

Reply:
Yes i agree I shall change it